src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -56,7 +56,7 @@ public async Task Page_SimpleForms_RenderAntiforgery()
             ResourceFile.UpdateFile(_resourcesAssembly, outputFile, expectedContent, responseContent);
 #else
             expectedContent = string.Format(expectedContent, forgeryToken);
-            Assert.Equal(expectedContent, responseContent.Trim(), ignoreLineEndingDifferences: true);
+            Assert.Equal(expectedContent, responseContent, ignoreLineEndingDifferences: true);
```

Comment:
I don't understand why this change was necessary

Reply:
This is directly related to the build break. If we trim the response and the expected text contains a leading newline, the test fails.

As I said in the description, trimming the responses is wasteful and incorrect. No reason for this in any of the functional tests using `GENERATE_BASELINES`.